src/inc/quic_platform_winkernel.h

```diff
@@ -30,6 +30,7 @@
 #pragma warning(disable:26071)
 #pragma warning(disable:28118)
 #pragma warning(disable:28196)
+#pragma warning(disable:28251)
```

Comment:
we probably shouldn't ignore this one, though. It tells us if our header/source annotations are in sync. Or it should.

Reply:
We're just ignoring this on public headers for the most part. We don't want to fail our build if the public SDK/WDK headers have issues (which is what is happening here).